crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -447,3 +470,43 @@ pub fn annotated_assignment_default_in_stub(
     }
     checker.diagnostics.push(diagnostic);
 }
+
+// PYI052
+pub fn assignment_default_in_async_stub(
```

Comment:
What's the relevance of "async" in these various names and contexts? I don't think Y052 has any relationship to async, but trying to understand.

Reply:
Oh I guess I got the naming wrong, I agree it does not have anything to do with async here.